resources/assets/js/app/components/Notifications.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {getTranslate} from 'react-localize-redux';
+import BroadcastService from 'app/services/BroadcastService';
+import {getMessage} from 'app/services/NotificationService';
+import {addNotification} from 'features/notifications/actions';
+import Push from 'push.js';
+import {browserHistory} from 'react-router';
+import { NotificationStack } from 'react-notification';
+
+class Notifications extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            notifications: []
+        };
+    }
+
+    componentWillMount() {
+        this.getNotifications(this.props);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.userId !== this.props.userId) {
+            this.getNotifications(nextProps);
+        }
+    }
+
+    getNotifications(props) {
+        const {addNotification, userId} = props;
+
+        if (userId > 0) {
+            BroadcastService.Echo.private('App.User.' + userId)
+                .notification((notification) => {
+                    const data = Object.assign(notification, {
+                        type: BroadcastService.prepareType(notification.type),
+                        created_at_x: parseInt(Date.now() / 1000),
+                        read: false
+                    });
+
+                    addNotification(data);
+                    this.showNotification(data);
+                });
+        }
+    }
+
+    showNotification(notificationData) {
+        const messageData = getMessage(notificationData),
+            notification = {
+                key: notificationData.id,
+                message: messageData.title,
+                title: 'BeBeep',
+                timeout: 5000,
+                onClick: () => {
+                    browserHistory.push('/dashboard/notifications')
```

Comment:
base on notification type navigate use to appropriate page like trip or booking page and mark this notification as read. For cases when page can't be opened - navigate to notification page